data_steward/cdr_cleaner/cleaning_rules/controlled_tier/concept_suppression.py

```diff
@@ -135,16 +135,10 @@ def get_query_specs(self, *args, **keyword_args) -> query_spec_list:
 
         return sandbox_queries + queries + delete_empty_sandbox_queries
 
-    def setup_validation(self, client, *args, **keyword_args):
-        pass
-
-    def validate_rule(self, client, *args, **keyword_args):
-        pass
-
```

Comment:
We should discuss these two and how you envision their usage.  Removing them from here does mean the extending class has to implement them.  That's not happening in `birth_information_suppression.py`.

Reply:
Totally agreed. I think we might've briefly talked about it. I remember myself saying we should all implement `validation_rule`. However,  I keep going back to the same doubt. Whenever I think of implementing a validation rule, I would almost end up with the same cleaning rule query (could be a slight modification of the main query such as adding select COUNT(*) around it), so if we use the same query that produces the cleaned version of the table to validate, it would not be a fair validation. 

In the meantime, I will add those two methods to `birth_information_suppression.py`